src/pages/Registration/Steps/getRegistrationState.ts

```diff
@@ -40,22 +40,23 @@ export function getRegistrationState(
   if (isDoneBanking(reg)) {
     const { ContactPerson: c, Registration: r } = reg;
     return {
-      step: 5,
+      step: 6,
```

Comment:
The steps were set wrongly:
- expected behavior -> navigate to the step after the most recently finished one
- actual (wrong) behavior -> navigates to the most recently finished step

Reply:
I see what you mean.

Maybe something like this would work https://github.com/AngelProtocolFinance/angelprotocol-web-app/pull/2506/commits/7b5b732c0a1d5b53b107a6f4eda5fe00d13710d9